app/src/components/main/image/image.jsx

```diff
@@ -97,7 +97,7 @@ export class Image extends Component {
   };
 
   normalizeFileTypeForSvg = (file) => {
-    if (file.type === 'text/xml') {
+    if (file.type === 'text/xml' || file.type === 'text/xml;charset=utf-8') {
```

Comment:
Please make it more flexible to support different charsets, use `file.type.indexOf('text/xml')` or smth like that.